app/models/operations/load_benefit_market_catalog.rb

```diff
@@ -68,13 +68,12 @@ def load_plans(input)
       parse_files(files)
       additional_files = Dir.glob(File.join(Rails.root, "db/seedfiles/plan_xmls/#{input[:state]}/master_xml", '**', '*.xlsx'))
 
-      parsed_files = parse_files(files)
-      parsed_additional_files = parse_files(additional_files)
+      parsed_files = { package_xml_files: parse_files(files) }
+      parsed_additional_files = { plan_xlsx_files: parse_files(additional_files) }
 
-      transaction = Transactions::LoadPlans.new
-      transaction.with_step_args(
-        load_file_info: [parsed_additional_files]
-      ).call(parsed_files)
+      transaction = Operations::LoadPlans.new
```

Comment:
since it's an operation now instead of transaction, can you rename it to operation.

Reply:
yeah sure